yaksh/models.py

```diff
@@ -1509,6 +1509,7 @@ def update_total_marks(self):
         for question_set in self.random_questions.all():
             marks += question_set.marks * question_set.num_questions
         self.total_marks = marks
+        self.save()
```

Comment:
The function is being used in only one [place](https://github.com/FOSSEE/online_test/blob/a7b4b9e0c6699987d1dee4007e9c62da15559432/yaksh/models.py#L1615) in the ```models.py``` file. And the question paper is externally saved over there.

This addition is currently redundant. Please either remove the external call to save or remove this addition.